boards/e180-zg120b-tb/doc.txt

```diff
@@ -0,0 +1,199 @@
+| EEPROM        | no                                                                                      |
+| Frequency     | up to 38.4 MHz                                                                          |
+| FPU           | yes                                                                                     |
+| MPU           | yes                                                                                     |
+| DMA           | 8 channels                                                                              |
+| Timers        | 2x 16-bit + 1x 16-bit (low power)                                                       |
+| ADCs          | 12-bit ADC                                                                              |
+| UARTs         | 2x USART, 1x LEUART                                                                     |
+| SPIs          | 2x USART                                                                                |
+| I2Cs          | 1x                                                                                      |
+| Vcc           | 1.85 V - 3.8 V                                                                          |
+| Datasheet     | [Datasheet](https://www.silabs.com/documents/public/data-sheets/efr32mg1-datasheet.pdf) |
+| Manual        | [Manual](https://www.silabs.com/documents/public/reference-manuals/efr32xg1-rm.pdf)     |
+| Board Manual  | [Board Manual](http://www.ebyte.com/en/downpdf.aspx?id=896)                             |
+
+### Peripheral mapping
+| Peripheral | Number  | Hardware        | Pins                        | Comments                                            |
+|------------|---------|-----------------|-----------------------------|-----------------------------------------------------|
+| ADC        | 0       | ADC0            | CHAN0: internal temperature | Ports are fixed, 14/16-bit resolution not supported |
+| HWCRYPTO   | &mdash; | &mdash;         |                             | AES128/AES256, SHA1, SHA256                         |
+| RTT        | &mdash; | RTCC            |                             | 1 Hz interval. Either RTT or RTC (see below)        |
+| RTC        | &mdash; | RTCC            |                             | 1 Hz interval. Either RTC or RTT (see below)        |
+| Timer      | 0       | TIMER0 + TIMER1 |                             | TIMER0 is used as prescaler (must be adjecent)      |
+|            | 1       | LETIMER0        |                             |                                                     |
+| UART       | 0       | USART0          | RX: PA1, TX: PA0            | Default STDIO output                                |
+
+### User interface
+| Peripheral | Mapped to | Pin  | Comments        |
+|------------|-----------|------|-----------------|
+| Button     | PB0_PIN   | PD15 | Mode Change     |
+|            | PB1_PIN   | PD13 | Touch Link      |
+|            | PB2_PIN   | PB11 | Baud Rate Reset |
```

Comment:
So the fourth button (with the Chinese characters) is not connected at all?

Reply:
That's the reset button. I used Google Lens to identify it :-)

Reply:
Probably worth adding a note that the fourth button is reset, wouldn't have guessed it